app/src/main/java/org/ieeeguc/ieeeguc/models/User.java

```diff
@@ -95,6 +98,64 @@ public String getCommitteeName() {
         return committeeName;
     }
 
+    public static void ForgetPassword(String email, final HTTPResponse HTTPR) {
+        HashMap <String,String>p = new HashMap();
+        p.put("email",email) ;
+        OkHttpClient client = new OkHttpClient();
+        final Request request = new Request.Builder()
+                .url("http://ieeeguc.org/api/login")
+                .post(RequestBody.create(contentType,( new JSONObject(p)).toString()))
+                .build();
+        client.newCall(request).enqueue(new Callback() {
+            public void onFailure(Call call, IOException e) {
+                e.printStackTrace();
+            }
+
+            @Override
+            public void onResponse(Call call, final Response response) {
+                try {
+                    HTTPR.onSuccess(200,new JSONObject(response.body().toString())) ;
+                } catch (JSONException e) {
+                    e.printStackTrace();
+                }
+            }
+        });
+    }
+
+    public static void login(String email , String password ,final HTTPResponse HTTP_RESPONSE){
+        final JSONObject jsonB = new JSONObject();
+        try{
+            jsonB.put("email",email);
+            jsonB.put("password",password);
+        }catch(JSONException e){
+            e.printStackTrace();

```

Comment:
you have to call `HTTPResponse.onFailure()` when your method fails.
Also you can't continue execution when the json object that you are trying to send is failed to be constructed